impl/client/src/main/javascript/web/util/_a11y.js

```diff
@@ -35,6 +35,415 @@ define(["./_focus"], function(focusUtil) {
     delete: 46
   };
 
+  function makeAccessibleSelect(elem) {
+    var listboxNode = elem;
+    listboxNode.setAttribute('role', 'listbox');
+    listboxNode.setAttribute('tabindex', '0');
+    listboxNode.setAttribute('aria-activedescendant', '');
+    listboxNode.setAttribute('aria-multiselectable', 'true');
+    var activeDescendant = listboxNode.getAttribute(
+        'aria-activedescendant'
+    );
+    var multiselectable = listboxNode.hasAttribute('aria-multiselectable');
+    var startRangeIndex = 0;
+    var keysSoFar = '';
+    var handleFocusChange = function () {};
+    var handleItemChange = function () {};
+    registerEvents();
+
+    /**
+     * @description
+     *  If there is no activeDescendant, focus on the first option
+     */
+    function setupFocus() {
+      activeDescendant = listboxNode.getAttribute(
+          'aria-activedescendant'
+      );
+      if (activeDescendant) {
+        var currentItem = document.getElementById(activeDescendant);
+        currentItem.classList.add('keyboard-navigation-focus');
+      }
```

Comment:
Here and elsewhere, it would be best to create a method such as `getActiveDescendant()` which would return the current active descendant or `null`, if none. Synchronizing the outer function scope' variable `activeDescendant`, here and elsewhere, but not always!, does not seem to provide any advantage. On the contrary, it brings confusion as to whether the variable value is expected to always be current or not.